File: Iv Layhort/Scala-Layhort/src/Task_4/Exercise_1_ComputeNet1.scala
package Task_4

import scala.io.StdIn._
object Exercise_1_ComputeNet1 {
  def main(args: Array[String]): Unit = {

    println("Enter the hourly rate : ")
    val hourlyRate = readDouble()
    println("Enter the hours work : ")
    val hoursWorked = readDouble()

    val grossPay = hourlyRate * hoursWorked
    val witholdingTax = if(grossPay<=300) grossPay*0.10 else grossPay*0.12
    val netPay = grossPay - witholdingTax

    println(s"The gross pay is : ${grossPay%.2f}")
    println(s"The witholding tax is : ${witholdingTax%.2f}")
    println(s"The net pay is : ${netPay%.2f}")
  }
}

File: Iv Layhort/Java_Layhort/src/Task_4/Exercise_1_ComputeNet1.java
package Task_4;
import java.util.*;
public class Exercise_1_ComputeNet1 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double hourlyPayRate,hoursWorked,grossPay,witholdingTax, netPay;
        System.out.println("Enter the hourly pay rate :");
        hourlyPayRate = input.nextDouble();
        System.out.println("Enter the hours worked : ");
        hoursWorked = input.nextDouble();

        grossPay = hourlyPayRate*hoursWorked;

        witholdingTax = grossPay<=300.00 ? grossPay* 0.10: grossPay*0.12;

        netPay = grossPay - witholdingTax;

        System.out.println("The gross pay is "+grossPay%.2f);
        System.out.println("The witholding tax is "+witholdingTax%.2f);
        System.out.println("The net pay is "+netPay%.2f);
    }
}
